rstudio-wsl/draft.Rmd

```diff
@@ -0,0 +1,170 @@
+
+> Installing, this may take a few minutes...                                                                                                             
+> 
+> Please create a default UNIX user account. The username does not need to match your Windows username.                                                  
+> For more information visit: `https://aka.ms/wslusers` 
+> 
+> Enter new UNIX username:                                                                                                                               
+  
+Provide a user name followed by password that will be used for both your WSL distribution and RStudio Server authentication later on.
+
+You can now close the WSL instance by typing the following command:
+   
+```{bash}
+exit
+```
+
+* **Note:** Unlike the PowerShell, you cannot use `Ctrl + V` shortcut key to paste the clipboard content into the WSL terminal. Alternatively, press mouse right-click button to emulate the paste function.
+
+## RStudio Server Setup {#rstudio-server-setup}
+
+Now that you have successfully installed a Linux distribution on your system via WSL, you can setup and run RStudio Server by following the procedures below:
+
+1. Press `Windows` logo key to open start menu, then type `wsl` to launch your installed Linux distro.
+1. Make sure your Linux packages up-to-date using the following command (we will assume that you have installed a Ubuntu distribution):
+
+    ```{bash}
+    sudo apt-get update
+    sudo apt-get upgrade -y
+    ```
+
+    * **Note:** Provide the password of your user account that you created for the Linux system, not the Windows password to log in for your PC.
+
+1. Add the following apt credential to your repository list  for Getting the Latest R (v4.0.0 as of this writing):
+
+    ```{bash}
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+    sudo add-apt-repository 'deb https://cloud.r-project.org/bin/Linux/ubuntu focal-cran40/'
+    ```
+
+    * **Note:** More detailed information on installing the latest R for Linux can be found at [the official CRAN page](https://cran.r-project.org/bin/Linux/ubuntu/){target="_blank"}.
+
+1. Install R with some essential dependencies for the RStudio Server and devtools:
+
+    ```{bash}
+    sudo apt install -y r-base r-base-core r-recommended r-base-dev gdebi-core build-essential libcurl4-gnutls-dev libxml2-dev libssl-dev
+    ```
+
+1. Install RStudio Server:
+
+    ```{bash}
+    wget https://rstudio.org/download/latest/stable/server/bionic/rstudio-server-latest-amd64.deb
+    sudo gdebi rstudio-server-latest-amd64.deb
+    ```
+
+    * **Note1:** You may receive a warning message like "Couldn't find an alternative telinit implementation to spawn." However, our testing results have revealed that there is no issue with using RStudio Server regardless of this warning.
```

Comment:
Instead of "However, our testing results have revealed that there is no issue with using RStudio Server regardless of this warning." let's just say "This is a known issue and can be safely ignored."